site/examples/spinner.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { HtmlExample } from '../html_example';
+
+const spinner = `<div class=' viewport-full relative clip'>
+  <div class='flex-parent flex-parent--center-cross flex-parent--center-main bg-darken10 viewport-twothirds viewport-full-ml'>
+    <div class='flex-child loading'></div>
```

Comment:
The advertised position is that you should put `flex-child` on every child of a `flex-parent`.